interface/forms/newGroupEncounter/common.php

```diff
@@ -279,7 +279,7 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
  if ($sensitivities && count($sensitivities)) {
   usort($sensitivities, "sensitivity_compare");
 ?>
-     <td class='bold' nowrap><?php echo xlt('Sensitivity'); ?>:</td>
+     <td class='bold' nowrap><?php echo xlt('Sensitivity:'); ?>:</td>
```

Comment:
nice thinking and very thoughtful to reduce constants for translators!
https://github.com/openemr/openemr/blob/master/contrib/util/language_translations/currentConstants.txt#L6715
(agreed that if the constant already exists with the colon, then to use it)